specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/Accounts.json

```diff
@@ -334,6 +334,61 @@
       },
       "type": "object"
     },
+    "ListEdgePoliciesInput": {
+      "properties": {
+        "deviceId": {
+          "type": "string",
+          "description": "Unique identifier of the edge device."
+        }
+      },
+      "type": "object"
+    },
+    "EdgeUsageDataEventHub": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the Event Hub where usage will be reported."
+        },
+        "namespace": {
+          "type": "string",
+          "description": "Namespace of the Event Hub where usage will be reported."
+        },
+        "token": {
+          "type": "string",
+          "description": "SAS token needed to interact with Event Hub."
+        }
+      },
+      "type": "object"
+    },
+    "EdgeUsageDataCollectionPolicy": {
+      "properties": {
+        "dataCollectionFrequency": {
+          "type": "string",
+          "description": "Data collection frequency."
```

Comment:
nit. Should describe the expected format given it is string type. Same below as well.